lib/util/prompt-suggestion.js

```diff
@@ -54,15 +54,21 @@ var getListDefault = function (question, defaultValue) {
  *
  * @param  {Object}       question Inquirer prompt item
  * @param  {String|Array} answer   The inquirer answer
+ * @param  {Boolean}      storeAll Should store default values
```

Comment:
@SBoudrias The original behaviour was not to store default values. I didn't want to break this. So storeAll set to false will reflect the original workflow. What do you suggest?